src/plone/recipe/zope2instance/recipe.py

```diff
@@ -698,7 +699,17 @@ def is_rs_option(name):
         if options.get("storage-wrapper"):
             storage_snippet = indent(options["storage-wrapper"] % storage_snippet, 4)
 
-        zodb_tmp_storage = options.get("zodb-temporary-storage", "on")
+        zodb_tmp_storage = options.get("zodb-temporary-storage")
+        if zodb_tmp_storage is None:
+            # What should be the default?
+            # In Plone 6 we do not want to use the temporary storage anymore.
+            # See https://github.com/plone/plone.recipe.zope2instance/issues/180
+            requirements, ws = self.egg.working_set(["plone.recipe.zope2instance"])
```

Comment:
Calculating the working set is presumably expensive. Is it cached in some way between here and `install_scripts`?

Reply:
Good question.

I timed it with `time.time()`. Getting the working set here takes 3.0 seconds. And then when installing the scripts it takes 0.007 seconds. So setuptools/buildout/whoever is doing the caching for us. :-)